nx.json

```diff
@@ -0,0 +1,54 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "lint",
+          "build",
+          "test",
+          "test:contracts",
+          "prettier",
+          "solhint",
+          "test:forge",
+          "test:hardhat",
+          "test:solidity",
+          "test:typescript",
+          "build:solidity",
+          "build:typescript"
```

Comment:
This is the list of operations nx should check cache for.    

For case of build it will automatically use cached outputs
In case of tests or lint it will just cache whether the tests passed or not